pyproject.toml

```diff
@@ -182,6 +182,7 @@ hiredis = { version = "*", optional = true }
 Pympler = { version = "*", optional = true }
 parameterized = { version = ">=0.7.4", optional = true }
 idna = { version = ">=2.5", optional = true }
+rust-python-jaeger-reporter = {git = "https://github.com/erikjohnston/rust-jaeger-python-client.git"}
```

Comment:
Just adding this extra dependency. Why do I get an error with `jaeger_client` which didn't change at all?

```
$ poetry run synapse_homeserver --config-path homeserver.yaml
Error during startup:
Traceback (most recent call last):
  File "/Users/eric/Documents/github/element/synapse/synapse/app/_base.py", line 242, in wrapper
    await cb(*args, **kwargs)
  File "/Users/eric/Documents/github/element/synapse/synapse/app/homeserver.py", line 390, in start
    await _base.start(hs)
  File "/Users/eric/Documents/github/element/synapse/synapse/app/_base.py", line 451, in start
    init_tracer(hs)  # noqa
  File "/Users/eric/Documents/github/element/synapse/synapse/logging/opentracing.py", line 430, in init_tracer
    config = JaegerConfig(
  File "/Users/eric/Library/Caches/pypoetry/virtualenvs/matrix-synapse-yTA0S3Fg-py3.9/lib/python3.9/site-packages/jaeger_client/config.py", line 111, in __init__
    if get_boolean(self.config.get('metrics', True), True):
AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
Ahh, it's probably because I had `jaeger_config:` uncommented but no fields defined in it